Source/UnitTests/UnitTests.vcxproj

```diff
@@ -50,7 +56,6 @@
     <ClCompile Include="Common\EventTest.cpp" />
     <ClCompile Include="Common\FileUtilTest.cpp" />
     <ClCompile Include="Common\FixedSizeQueueTest.cpp" />
-    <ClCompile Include="Common\FlagTest.cpp" />
```

Comment:
Why did you remove this?

Reply:
Not a clue how this one happened O_o It didn't show up in Visual Studio as a change either.